config/samples/default.yaml

```diff
@@ -8,4 +8,24 @@ metadata:
   name: default
   namespace: kyma-system
 spec:
-  # TODO(user): Add fields here
+  labels: {}
+  annotations: {}
+  companionConfig:
+    namespace: ai-core
+    configMapNames:
+      - ai-backend-config
+    secretNames:
```

Comment:
would it make more sense to split this like so?:

```
spec:
   secrets:
      ai-core: "ai-core"
      registry: "ai-ghcr-login-secret"
```

Reply:
I also like in general this suggestion [here](https://github.com/kyma-project/kyma-companion/issues/59#issuecomment-2247305664)